.github/workflows/rust.yml

```diff
@@ -27,73 +27,108 @@ jobs:
     runs-on: ubuntu-latest
     needs: [get_projectserum_version]
     container:
```

Comment:
should this be commented out too?

Reply:
Good catch - commented them all out. 9c551f9